packages/components/inputs/search-select-input/src/search-select-input.spec.js

```diff
@@ -0,0 +1,327 @@
+      const onChange = jest.fn();
+      renderInput({
+        onChange,
+      });
+      const input = screen.getByLabelText('Fruit');
+      fireEvent.focus(input);
+      fireEvent.keyDown(input, { key: 'ArrowDown' });
+      fireEvent.change(input, { target: { value: 'mango' } });
+      await waitFor(() => screen.getByText('Mango'));
+      screen.getByText('Mango').click();
+      expect(onChange).toHaveBeenCalledWith({
+        persist: expect.any(Function),
+        target: {
+          name: 'some-name',
+          value: { value: 'mango', label: 'Mango' },
+        },
+      });
+    });
+  });
+});
+
+describe('in multi mode', () => {
+  describe('when no value is specified', () => {
+    it('should render a select input', () => {
+      renderInput({
+        isMulti: true,
+        value: [],
+      });
+      const input = screen.getByLabelText('Fruit');
+      expect(input).toBeInTheDocument();
+    });
+    describe('when values are specified', () => {
+      it('should render a select input with preselected values', () => {
+        renderInput({
+          isMulti: true,
+          value: [
+            { value: 'mango', label: 'Mango' },
+            { value: 'raspberry', label: 'Raspberry' },
+          ],
+        });
+        const input = screen.getByLabelText('Fruit');
+        expect(input).toBeInTheDocument();
+        expect(screen.getByText('Mango')).toBeInTheDocument();
+        expect(screen.getByText('Raspberry')).toBeInTheDocument();
+      });
+    });
+  });
+  describe('interacting', () => {
+    it('should be able to select two option', async () => {
+      renderInput({
+        isMulti: true,
+        value: [],
+      });
+      const input = screen.getByLabelText('Fruit');
+      fireEvent.focus(input);
+      fireEvent.keyDown(input, { key: 'ArrowDown' });
+      fireEvent.change(input, { target: { value: 'mango' } });
+      await waitFor(() => screen.getByText('Mango'));
+      screen.getByText('Mango').click();
+      // new selected value should be Mango
+      expect(screen.getByText('Mango')).toBeInTheDocument();
+      // open list again
+      fireEvent.focus(input);
+      fireEvent.keyDown(input, { key: 'ArrowDown' });
+      fireEvent.change(input, { target: { value: 'banana' } });
+      await waitFor(() => screen.getByText('Banana'));
+      screen.getByText('Banana').click();
+      // new values should be Banana and Mango
+      expect(screen.getByText('Banana')).toBeInTheDocument();
+      expect(screen.getByText('Mango')).toBeInTheDocument();
+    });
+    it('should call onChange when two values selected', async () => {
+      const onChange = jest.fn();
+      renderInput({
+        onChange,
+        isMulti: true,
+        value: [],
+      });
+      const input = screen.getByLabelText('Fruit');
+      fireEvent.focus(input);
+      fireEvent.keyDown(input, { key: 'ArrowDown' });
+      fireEvent.change(input, { target: { value: 'mango' } });
```

Comment:
I think eventually ui-kit should evaluate using `user-event` for newer tests which I think is now exposed from testing-library-react.

Reply:
I checked the `package.json` for the repo and we use `@testing-library/react` which does not export the `user-event` helper. We have to add `@testing-library/user-event` to the dev-dependencies and the update the `test-utils` helper in the ui-kit to export those via `export { default as userEvent } from '@testing-library/user-event';`

Reply:
Thanks for checking. I think it's worthwhile as a follow-up then? Like add userEvent and migrate it all? I often feel - if scope permits - a good Friday afternoon task :)

Reply:
I think we should add it to ui-kit jira board and do it after the `readme` migration. Otherwise we might loose track of things. i.e. status of readme migration and migrating to user-events